src/notice/notice_backtrace.rs

```diff
@@ -0,0 +1,141 @@
+use crate::backtrace::{Backtrace, BacktraceFrame, BacktraceSymbol};
+use serde::ser::{Serialize, SerializeSeq, Serializer};
+use serde_json::{self, Value};
+use std::collections::HashMap;
+
+#[derive(Debug, Clone)]
+pub struct NoticeTrace {
+    frames: Vec<NoticeFrame>,
+}
```

Comment:
Maybe resist this model. Creating a model around Vec<NoticeFrame> was done mostly for implementing From<Backtrace>, but if we can do `impl From<&Backtrace> for Vec<NoticeFrame>` then we can completely do away with this semi-redundant model.

Reply:
Refactored away this model 👍